homeassistant/components/honeywell/climate.py

```diff
@@ -126,31 +129,36 @@ def __init__(self, client, device, cool_away_temp,
         self._username = username
         self._password = password
 
-        self._supported_features = (SUPPORT_PRESET_MODE |
-                                    SUPPORT_TARGET_TEMPERATURE_RANGE)
+        self.entity_id = "{}.{}".format(DOMAIN, device.deviceid)
```

Comment:
Should I put this in? People will now have a unique, persistent ID, but they will have to search-and-replace entity IDs in automations, etc.